articles/extensions/delegated-admin/hooks.md

```diff
@@ -191,24 +191,37 @@ Supported action names:
  - `unblock:user`
  - `send:verification-email`
 
-#### Create Hook
+#### Write Hook
 
-Whenever new users are created you'll want these users to be assigned to the group/department/vendor/... of the current user. This is what the **Create Hook** allows you to configure.
+Whenever new users are created you'll want these users to be assigned to the group/department/vendor/... of the current user. This is what the **Write Hook** allows you to configure.
 
 Hook contract:
 
  - `ctx`: The context object.
+   - `request.originalUser`: The current user values, payload is the new set of fields.
    - `payload`: The payload object.
      - `memberships`: An array of memberships that were selected in the UI when creating the user.
      - `email`: The email address of the user.
      - `password`: The password of the user.
      - `connection`: The name of the user.
+   - `userFields`: The user fields array, if one is specified in the settings query.
```

Comment:
Provide link to settings query (#the-settings-query-hook) since it isn't defined or anything 'til later on in the page?